email-common/src/main/java/io/commercetools/sunrise/email/MessageEditor.java

```diff
@@ -0,0 +1,21 @@
+package io.commercetools.sunrise.email;
+
+import javax.annotation.Nonnull;
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+
+/**
+ * A functional interface for modifying a message without needing to handle checked exceptions like the
+ * {@link MessagingException}s thrown by {@link MimeMessage}.
+ */
+@FunctionalInterface
+public interface MessageEditor {
```

Comment:
So it would be easier to write a POJO which can be converted to a MimeMessage.

It sounds for me like we should have used Apache Email which already has that kind of bean.